app/models/miq_report/formatting.rb

```diff
@@ -97,7 +97,7 @@ def format(col, value, options = {})
     options[:column] = col
 
     # Chargeback Reports: Add the selected currency in the assigned rate to options
-    if db.to_s == "ChargebackVm" || db.to_s == "ChargebackContainerProject"
+    if Chargeback.db_is_chargeback?(db)
```

Comment:
I'm find with the autoload (much better than class.to_s checks), but the first question is important to understand.